examples/panorama/variables.tf

```diff
@@ -1,18 +1,105 @@
 # General
-variable "project" {}
-variable "project_id" {}
-variable "region" {}
+variable "project" {
+  description = "The project name to deploy the infrastructure in to."
+  type        = string
+  default     = null
+}
+variable "region" {
+  description = "The region into which to deploy the infrastructure in to"
+  type        = string
+  default     = "us-central1"
+}
 
 # VPC
-variable "vpc_name" {}
-variable "subnet_name" {}
-variable "cidr" {}
-variable "allowed_sources" {}
+variable "vpcs" {
+  description = <<-EOF
+    A map containing each network setting:
+
+    Available options :
```

Comment:
For this kind of details let's maybe add a link directly to module input description?